c2corg_ui/templates/base.html

```diff
@@ -149,6 +151,10 @@
            vcRecaptchaServiceProvider.setSiteKey('${settings["recaptcha_site_key"]}');
          });
 
+         angular.module('app').constant('angularMomentConfig', ['moment-timezone'], {
+           timezone: 'Europe/London'
```

Comment:
Because I think that I had some problems on Thursday and this was the latest WIP commit where I tested different settings